### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by meejle2**
# Description
criticalchicken

# Copilot

### Summary
📝🎨📦

Added a new applet called `criticalchicken` that shows the current status of the Critical Role Kickstarter campaign. Created the necessary metadata files (`critical_chicken.star` and `manifest.yaml`) for the applet to run on the Tidbyt device and app and to be published on the community repository.

> _`critical_chicken`_
> _Metadata for applet and repo_
> _A winter project_

### Walkthrough
*  Define and specify applet metadata for Critical Chicken app ([link](https://github.com/tidbyt/community/pull/1729/files?diff=unified&w=0#diff-c114c3ea5a8758e5c4f81c870c641a43db6b0b4fe38c8d9c4442f8d4c66b73feR1-R369), [link](https://github.com/tidbyt/community/pull/1729/files?diff=unified&w=0#diff-dd4ba1e8f8b8307d5617ffa529237e46db9f15c1e4741cd839ae9668033ead5cR1-R8))

### CODE REVIEW

**Comment by meejle2** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by meejle** [comment]
> Added a new applet called criticalchicken that shows the current status of the Critical Role Kickstarter campaign.

Lmao where did it get that idea from? 🤔 😅

File: apps/criticalchicken/critical_chicken.star [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+
+# Sorry...
+imgSorry = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAZklEQVQ4jdVUOQ7AIAwLeTHmIeTLHVxZEaIdOkDxgmNZwglHQa12MpxLj+gR4lNxMKjcBQZwsgY0QFzhnvhGaHwM7PaaTL7M6Ze+GMO+LnXaw29PQLiv0Ofp5vewpr2c08zK6b/QBeMeUr9B6cGnAAAAAElFTkSuQmCC
+""")
+
+# MAIN
+
+def main():
+    headline_cached = cache.get("cached_headline")
```
Comment (matslina) [inline apps/criticalchicken/critical_chicken.star:201]: Check out the built in cache functionality of the http module. Just pass`ttl_seconds` to `http.get()` and you get caching for free. Easier than using the cache module, and also more future proof as we may change the behavior and scope of the cache module in the future.

**Review by matslina — CHANGES_REQUESTED** [review]
Thanks for the PR!

We'll need all files in a single directory to merge. Would probably suggest not including all the PNGs in the PR.

**Pull request closed without merging**